File: sentimental-mario-less/mario.py
from cs50 import get_int

# Asking the user for input
while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

for i in range(n):
    # Printing spaces
    for j in range(n-1-i):
        print(" ", end="")
    # Printing hashes
    for k in range(i+1):
        print("#", end="")
    print()

File: mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    // number of rows
    for (int i = 1; i < n + 1; i++)
    {
        for (int s = 0; s < n - i; s++)
        {
            printf(" "); // print spaces
        }
        for (int j = 0; j < i ; j++)
        {
            printf("#"); // print hashes
        }
        {
            printf("  "); //space between
        }
        for (int j = 0; j < i ; j++)
        {
            printf("#"); // print hashes
        }

        printf("\n"); // print next line
    }
}